python/vulture/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                vulture
```

Comment:
Most things in the "python" category start with "py-". BTW, I don't see any categories listed (I'd say devel in addition to python.)

Reply:
I will add the category devel (or maybe even make devel the main category).

The port does not start with "py-" because it is not a Python module, but a program written in Python.
This is a MacPorts convention: indeed, if the port name starts with "py-", the python PortGroup creates the pyXX-* subports automatically and links binaries in ${prefix}/bin with the Python version as suffix, otherwise it does not create the subports and links the binaries without the suffix.
Compare, e.g., the ports frescobaldi, dblatex, mercurial, just to name some.

Reply:
> The port does not start with "py-" because it is not a Python module, but a program written in Python.

Ah! I utterly missed that! This changes a lot.